pkg/idtools/idtools_windows.go

```diff
@@ -1,18 +1,9 @@
-package idtools // import "github.com/docker/docker/pkg/idtools"
+package idtools
 
 import (
 	"os"
 )
 
-const (
-	SeTakeOwnershipPrivilege = "SeTakeOwnershipPrivilege"
-)
```

Comment:
As we're keeping this file, perhaps we could keep the consts here was well, instead of moving them to a separate file?